deploy/crds/operator.tigera.io_amazoncloudintegrations_crd.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
It's unfortunate that it uses a different name for the CRD. Let me see if I can trick git into treating this as a "moved" file.